bindgen/clang.rs

```diff
@@ -1493,6 +1493,15 @@ impl Type {
         }
     }
 
+    /// For atomic types, get the underlying type.
+    pub(crate) fn atomic_value_type(&self) -> Type {
+        unsafe {
+            Type {
+                x: clang_Type_getValueType(self.x),
```

Comment:
my C++ knowledge is almost null but what does this function return if the type is not an `AtomicType`?

Reply:
A CXType_Invalid, see [here](https://searchfox.org/llvm/rev/60ed1043d76554a48f66fc0c15fb97ac8b3af7b0/clang/tools/libclang/CXType.cpp#1354). It's very poorly named.